packages/in_app_purchase/lib/src/in_app_purchase_connection/in_app_purchase_connection.dart

```diff
@@ -7,6 +7,118 @@ import 'dart:io';
 import 'app_store_connection.dart';
 import 'google_play_connection.dart';
 import 'product_details.dart';
+import 'package:flutter/foundation.dart';
+
+/// Represents the data that is used to verify purchases.
+///
+/// The property [source] helps you to determine the method to verify purchases.
+/// Different source of purchase has different methods of verifying purchases.
+///
+/// Both platforms have 2 ways to verify purchase data. You can either choose to verify the data locally using [localVerificationData]
```

Comment:
Can you mention here that it's preferable for both platforms to verify them using a server if at all possible?